src/content/api-reference/index.ts

```diff
@@ -10,15 +9,17 @@ import { checkbox } from "./checkbox";
 import { collapsible } from "./collapsible";
 import { contextMenu } from "./context-menu";
 import { dateField } from "./date-field";
-import { dateRangeField } from "./date-range-field";
 import { datePicker } from "./date-picker";
+import { dateRangeField } from "./date-range-field";
 import { dateRangePicker } from "./date-range-picker";
+import { dialog } from "./dialog";
 import { dropdownMenu } from "./dropdown-menu";
 import { label } from "./label";
 import { linkPreview } from "./link-preview";
-import { popover } from "./popover";
-import { pagination } from "./pagination";
 import { menubar } from "./menubar";
+import { pagination } from "./pagination";
```

Comment:
just sorted these alphabetically in this file and others, revert if you think it's not needed